test/helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var expect = require('must');
+var STANDARD_DELAY = 200; // 200ms between nested calls to the API so things settle
```

Comment:
The delay allows the Github's filesystem to settle. Before I added the delay I found that the tests that created/updated files/repos failed nondeterministically because Github's system wasn't always fast enough to reflect how we were setting the tests up.

I suppose we could enable it only for writes if you want.